src/DataProtection/Cryptography.Internal/src/CryptoUtil.cs

```diff
@@ -76,12 +76,23 @@ public static Exception Fail(string message)
 #endif
         public static bool TimeConstantBuffersAreEqual(byte* bufA, byte* bufB, uint count)
         {
+#if NETSTANDARD2_0 || NET461
```

Comment:
Why is this block still needed❔ Might need comments before these preprocessor instructions explaining the restriction.

System.Security.Cryptography.Algorithms supports TFMs going back to `net46` and `netstandard1.3`. So, is this actually about `ReadOnlySpan<byte>`❔

Reply:
Perhaps, I'm not familiar with these apis at all, but I'll defer to @GrabYourPitchforks on the specifics, there's only Span versions of this api I believe

Reply:
Note: intelligence also claims that the AesGcm class is only available in net6.0 and not in net461/netstandardwhich is why these blocks were everywhere

Reply:
Consider inverting the check. 

```
#if NETCOREAPP
...
#else
...
```

Reply:
Done